packages/interfaces/src/dns_discovery.ts

```diff
@@ -46,6 +36,11 @@ export interface DnsDiscOptions {
    * Cause the bootstrap peer tag to be removed after this number of ms (default: 2 minutes)
    */
   tagTTL?: number;
+
+  /**
+   * Pass a custom DNS Client, to override DNS Parameters
+   */
+  dnsClient?: DnsClient;
```

Comment:
is it only for testing? 
I can't see it being used by consumers or am I missing?

Reply:
I don't think there is harm in it as it would enable a consumer to set their own DNS servers. But happy to find a better way if you prefer.